doc/config.rst

```diff
@@ -1905,6 +1905,15 @@ Options for the linkcheck builder
 
    .. versionadded:: 1.2
 
+.. confval:: linkcheck_anchors_ignore
```

Comment:
Sure, wasn't clear to me exactly how to add a test for this, but I'll dig and and can discuss further in the separate PR anyway.